modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -27,5 +28,10 @@
   "product.version" : "${product.version}",
 
   "transport.http.enabled" : true,
-  "transport.https.enabled" : true
+  "transport.https.enabled" : true,
+
+  "system.parameter.\u0027org.wso2.CipherTransformation\u0027": "AES/GCM/NoPadding",
```

Comment:
why we need to add a unicode here

Reply:
In toml keys are seperated by full stop (.)
Since org.wso2.CipherTransformation is one key, we need to define it as one key. We can use "system.parameter.'org.wso2.CipherTransformation'" or "system.parameter.\u0027org.wso2.CipherTransformation\u0027" . Since we have used unicode based one in other places [1], used the same.

[1]https://github.com/wso2/carbon-kernel/blob/e510cc9334f516d02b27aef3ddb670bf13ba45e9/distribution/kernel/carbon-home/repository/resources/conf/default.json#L174